sdk/src/main/java/com/horizen/utils/TimeToEpochUtils.scala

```diff
@@ -1,49 +1,56 @@
 package com.horizen.utils
```

Comment:
This was just an auto-reformat with scalafmt. If pure whitespace changes like these are undesirable I can undo this.